src/Compilers/CSharp/Test/Semantic/Semantics/OverloadResolutionPerfTests.cs

```diff
@@ -287,7 +287,8 @@ static class Ext
             comp.VerifyDiagnostics();
         }
 
-        [ConditionalFact(typeof(IsRelease))]
+        // Too slow to run with IOperationValidation on.
```

Comment:
Really? There are no unary or binary operators in this test, what caused the time it takes to increase?

Reply:
will try reactivating to see if it passes.